packages/esm-patient-list-app/src/patientListData/api.ts

```diff
@@ -84,33 +91,27 @@ export async function getPatientListsForPatient(patientUuid: string) {
   }: {
     results: Array<OpenmrsCohortMember>;
     error: Error;
-  } = await (await fetch(`/openmrs/ws/rest/v1/cohortm/cohortmember?patient=${patientUuid}&v=default`)).json();
+  } = await (
```

Comment:
`await await`?

Reply:
(the data is already resolved - no need for the outer await)

Reply:
Sure! I'll remove the outer one.